src/error.rs

```diff
@@ -1,27 +1,32 @@
-pub type Result<T> = ::std::result::Result<T, Error>;
+use std::sync::mpsc::SendError;
+use xcb::{Atom, base::{ConnError, GenericError}};
+use std::fmt;
 
-#[derive(Debug, Fail)]
 #[must_use]
+#[derive(Debug)]
 pub enum Error {
```

Comment:
implement [std::error::Error](https://doc.rust-lang.org/stable/std/error/trait.Error.html)?

Reply:
Yeah, I'll noticed that, too. I'll fix it.